cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -52,6 +52,7 @@ import org.ossreviewtoolkit.cli.utils.readOrtResult
 import org.ossreviewtoolkit.cli.utils.writeOrtResult
 import org.ossreviewtoolkit.evaluator.Evaluator
 import org.ossreviewtoolkit.model.FileFormat
+import org.ossreviewtoolkit.model.ResolvedPackageCurations.Companion.REPOSITORY_CONFIGURATION_PROVIDER_ID
```

Comment:
commit-message: Please describe the change in behavior and the reason why this change is desired.